can-validate.js

```diff
@@ -1,35 +1,6 @@
-var Construct = require('can-construct');
-var validate = require('can-validate/lib/validate');
-var helpers = require('can-validate/lib/helpers');
+var formatErrors = require('can-validate/format-errors/');
```

Comment:
I probably wouldn't have another module if this is all `can-validate` exports.  I'd move this code into `can-valdiate.js`